services/src/unit-test/java/uk/gov/moj/sdt/services/BulkSubmissionServiceTest.java

```diff
@@ -46,16 +20,12 @@
 import uk.gov.moj.sdt.domain.ServiceRouting;
 import uk.gov.moj.sdt.domain.ServiceType;
 import uk.gov.moj.sdt.domain.TargetApplication;
-import uk.gov.moj.sdt.domain.api.IBulkCustomer;
-import uk.gov.moj.sdt.domain.api.IBulkSubmission;
-import uk.gov.moj.sdt.domain.api.IErrorMessage;
+import uk.gov.moj.sdt.domain.api.*;
```

Comment:
Replace the wildcard import with the individual imports.  May need to change the IntelliJ import wildcard settings (https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports) to stop it doing this automatically.